crates/core/src/partials/mod.rs

```diff
@@ -1,3 +1,36 @@
+//! # Using partials
+//! To use `{% include %}` or `{% render %}` tags in a template, you first need to compile these
+//! included files as template partials.
+//!
+//! Example:
+//!
+//! ```liquid
+//! # common.liquid
+//! Number: {{ i }}
+//! ```
+//!
+//! ```rust,no_run,compile_fail
+//! // Build template partials using an eager, in-memory source compiler.
+//! // Other compilation policies also exist depending on specific needs.
+//! type Partials = EagerCompiler<InMemorySource>;
```

Comment:
These types (along with ParserBuilder, Parser, etc) are defined outside of this crate in a reverse dependent way, and I can't figure out what the right `# use ...` directive to use to make this snippet compile.